src/main/java/com/checkout/apm/oxxo/OxxoClient.java

```diff
@@ -0,0 +1,11 @@
+package com.checkout.apm.oxxo;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface OxxoClient {
+
+    CompletableFuture<Void> succeed(String id);
+
+    CompletableFuture<Void> expire(String id);
+
```

Comment:
Please change both parameters to `paymentId` to match the implementation class (and be more informative).